src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/LanguageServices/VisualBasicTypeInferenceService.TypeInferrer.vb

```diff
@@ -194,8 +195,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return SpecializedCollections.EmptyEnumerable(Of TypeInferenceInfo)
             End Function
 
-            Private Function InferTypeInTupleExpression(tupleExpression As TupleExpressionSyntax,
-                                                        argument As SimpleArgumentSyntax) As IEnumerable(Of TypeInferenceInfo)
+            Private Function InferTypeInTupleExpression(
+                    tupleExpression As TupleExpressionSyntax,
+                    previousToken As SyntaxToken) As IEnumerable(Of TypeInferenceInfo)
+                If previousToken = tupleExpression.OpenParenToken Then
+                    Return InferTypeInTupleExpression(tupleExpression, tupleExpression.Arguments(0))
```

Comment:
Do we need any tests for broken code cases? My gut is saying no because the parser would parse as a parenthesized expression in various cases but I could be very, very wrong.

Reply:
> My gut is saying no because the parser would parse as a parenthesized expression

In general, we would not. The typeinferrer is all about best-effort trying to figure out waht's going on.  It's rare that we do something particular bad in error cases (at worst, we just infer object or some other reasonable type).  Instead, we hear about broken cases we could do better in, and we put the tweak to this system to do that.